source/extensions/stat_sinks/metrics_service/grpc_metrics_proto_descriptors.cc

```diff
@@ -13,12 +14,22 @@ namespace MetricsService {
 
 void validateProtoDescriptors() {
   // https://github.com/envoyproxy/envoy/issues/9639
-  const envoy::service::metrics::v2::StreamMetricsMessage _dummy_v2;
+  const envoy::service::metrics::v2::StreamMetricsMessage _dummy_service_v2;
```

Comment:
Not sure how this missed from API boost and kept v2 without API_NO_BOOST macros, @htuch? But I think we should add them here or boost to v3.

Reply:
We can add API_NO_BOOST. But we want to keep the v2 dummys here (sorry if I am misunderstanding your "boost to v3" comment).

Reply:
Yeah lets add API_NO_BOOST here, or API_NO_BOOST_FILE comment on top of the file.

Reply:
@lizan updated